active/af1-identify-phi.py

```diff
@@ -26,8 +26,11 @@
 
 def generate_job_id(**kwargs):
     # get last update date from last completed run
-    tgt_select_stmt = "SELECT max(HDCPUpdateDate) FROM {table} WHERE job_status = %s".format(table=common_variables.AF1_RUNS)
-    update_date_from_last_run = common_hooks.AIRFLOW_NLP_DB.get_first(tgt_select_stmt, parameters=(common_variables.JOB_COMPLETE,))[0]
+    tgt_select_stmt = "SELECT max(HDCPUpdateDate) FROM {table} WHERE job_status = %s".\
+                      format(table=common_variables.AF1_RUNS)
+    update_date_from_last_run = common_hooks.AIRFLOW_NLP_DB.\
+                      get_first(tgt_select_stmt, parameters=(common_variables.JOB_COMPLETE,))[0].\
+                      strftime(common_variables.DT_FORMAT)[:-3]
```

Comment:
i'm a little confused, where is the hard code for dates earlier than 2020-01-09?

Reply:
per our discussion offline, we are hot fixing it in the pod for now.